osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -157,7 +155,7 @@ private double calculateStarRating(double basePerformance)
             if (basePerformance <= 0.00001)
                 return 0;
```

Comment:
Based off of [Frost's paper](https://docs.google.com/document/d/10DZGYYSsT_yjz2Mtp6yIJld0Rqx4E-vVHupCqiM4TNI/edit?tab=t.0#heading=h.wl20xe40vwy1) and my own understanding, I'm pretty sure that this isn't required anymore.

```
cbrt(1.14) * 0.0265 * (cbrt(100000 / 2^(1/1.1) * 0) + 4) = 0.1107322562431658
cbrt(1.12 * 0) = 0